bigtable/client/row_set.cc

```diff
@@ -20,6 +20,11 @@ namespace btproto = ::google::bigtable::v2;
 
 RowSet RowSet::Intersect(bigtable::RowRange const& range) const {
   RowSet result;
+  // Special case: "all rows", return the argument range
+  if (row_set_.row_keys().empty() and row_set_.row_ranges().empty()) {
+    return RowSet(range);
+  }
+  // Normal case
```

Comment:
Consider `// Normal case: find the intersection with row keys and row ranges in the RowSet`.